### PR TITLE
update compute dist

### PR DESCRIPTION
**Pull request by awirb**
* update naming of plots, logs 
* add write and read cfg function saved to Outputs

### CODE REVIEW

File: avaframe/in1Data/computeFromDistributionCfg.ini [hunk 1]
```diff
@@ -17,3 +17,5 @@ support = 10000
 name = scenario70
 # if min max value shall be included true, default False
 flagMinMax = False
+# name of distribution
```
Comment (fso42) [inline avaframe/in1Data/computeFromDistributionCfg.ini:20]: please list the possible options in the comment ie. [Pert, Uniform...] or whatever

File: avaframe/in3Utils/cfgUtils.py [hunk 2]
```diff
@@ -159,3 +160,38 @@ def compareConfig(iniFile, modName, compare):
...
+    # set outputs
+    outDir = os.path.join(avaDir, 'Outputs')
+
+    # write to file
+    with open(os.path.join(outDir, '%s%s_settings.ini' % (modName, suffix)), 'w') as conf:
+        cfg.write(conf)
+
+
+
+def readCfgFile(avaDir, module, suffix=''):
```
Comment (fso42) [inline avaframe/in3Utils/cfgUtils.py:182]: This function is not really needed as far as I can see, as the function: `cfgUtils.getModuleConfig` 
allows for a fileOverride in which a specified file can be used.

File: avaframe/in3Utils/cfgUtils.py [hunk 3]
```diff
@@ -11,6 +11,7 @@
 # Local imports
 import avaframe as avaf
 from avaframe.in3Utils import logUtils
+import pickle
```
Comment (fso42) [inline avaframe/in3Utils/cfgUtils.py:14]: Not needed?

File: avaframe/out3Plot/plotUtilsCfg.ini [hunk 4]
```diff
@@ -45,5 +45,5 @@ unitrelTh = m
 # name for result parameters
 nameppr = peak pressure
 namepfd = peak flow depth
-namepv = peak speed
+namepv = peak flow velocity
```
Comment (fso42) [inline avaframe/out3Plot/plotUtilsCfg.ini:48]:
```suggestion
namepv = peak velocity
```

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/in3Utils/cfgUtils.py (hunk 2 above)
Comment (awirb) [reply avaframe/in3Utils/cfgUtils.py:182]: I would suggest to keep it as a separate function as the writeCfgFile creates one that is called moduleName_settings.ini, and the read by default reads the moduleName_settings.ini so it appends the correct ending and I think in this way it is less likely to mix it with reading the standard/default moduleNameCfg.ini file..

File: avaframe/in3Utils/cfgUtils.py (hunk 3 above)
Comment (awirb) [reply avaframe/in3Utils/cfgUtils.py:14]: True, thanks

File: avaframe/in1Data/computeFromDistributionCfg.ini (hunk 1 above)
Comment (awirb) [reply avaframe/in1Data/computeFromDistributionCfg.ini:20]: Done

File: avaframe/out3Plot/plotUtilsCfg.ini (hunk 4 above)
Comment (awirb) [reply avaframe/out3Plot/plotUtilsCfg.ini:48]: keep this for now

**Review by fso42 — APPROVED** [review]

**Pull request merged**